src/code/V3ServerAPICalls.cs

```diff
@@ -284,38 +284,24 @@ public override FindResults FindVersionWithTag(string packageName, string versio
         /**  INSTALL APIS **/
 
         /// <summary>
-        /// Installs specific package.
-        /// Name: no wildcard support.
-        /// Examples: Install "Newtonsoft.json"
-        /// </summary>
-        public override Stream InstallName(string packageName, bool includePrerelease, out ErrorRecord errRecord)
-        {
-            _cmdletPassedIn.WriteDebug("In V3ServerAPICalls::InstallName()");
-            return InstallHelper(packageName, version: null, out errRecord);
-        }
-
-        /// <summary>
-        /// Installs package with specific name and version.
+        /// Installs a specific package.
         /// Name: no wildcard support.
-        /// Version: no wildcard support.
-        /// Examples: Install "Newtonsoft.json" -Version "1.0.0.0"
-        ///           Install "Newtonsoft.json" -Version "2.5.0-beta"
+        /// Examples: Install "PowerShellGet"
+        ///           Install "PowerShellGet" -Version "3.0.0"
         /// </summary>
-        public override Stream InstallVersion(string packageName, string version, out ErrorRecord errRecord)
+        public override Stream InstallPackage(string packageName, string packageVersion, bool includePrerelease, out ErrorRecord errRecord)
         {
-            _cmdletPassedIn.WriteDebug("In V3ServerAPICalls::InstallVersion()");
-            if (!NuGetVersion.TryParse(version, out NuGetVersion requiredVersion))
+            Stream results = new MemoryStream();
```

Comment:
Perhaps we can add a debug statement for this method, not blocking.
```suggestion
            Stream results = new MemoryStream();
            _cmdletPassedIn.WriteDebug("In V3ServerAPICalls::InstallPackage()");
```